packages/edit-widgets/src/blocks/legacy-widget/edit/handler.js

```diff
@@ -172,9 +172,9 @@ class LegacyWidgetEditHandler extends Component {
 
 		if ( widgetClass ) {
 			apiFetch( {
-				path: `/__experimental/widget-utils/form/${ encodeURIComponent(
+				path: `/wp/v2/widget-types/${ encodeURIComponent(
 					widgetClass
```

Comment:
```suggestion
					idBase
```
Like this @TimothyBJacobs ?

Reply:
I believe so, but we might also need to fall back to `id` for reference widgets.